dune-project

```diff
@@ -15,7 +15,7 @@
   (description "`function%distr A x, _ | _, A x when p x -> e` will result in `function A x, _ when p x -> e | _, A x when p x -> e`")
   (depends
    (dune (>= 2.3.0))
-   (ocaml-migrate-parsetree (>= 1.6.0))
+   ppxlib
```

Comment:
Should we specify any version constraint?

Reply:
I don't think that would be necessary. A lot of ppx rewriter folks like to avoid a dependency on `base` and `stdio` though. For that you'd need `ppxlib >= 0.15.0`.